src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/OverrideEqualsAndOperatorEqualsOnValueTypesTests.cs

```diff
@@ -387,6 +387,16 @@ End Structure
                 GetBasicOverrideEqualsDiagnostic(2, 18, "A"));
         }
 
+        [Fact, WorkItem(2324, "https://github.com/dotnet/roslyn-analyzers/issues/2324")]
+        public void CSharp_RefStruct_NoDiagnostic()
+        {
+            VerifyCSharp(@"
+public ref struct A
+{
+    public int X;
+}", parseOptions: new CSharpParseOptions(LanguageVersion.CSharp8));
```

Comment:
I don't know what's best between `CSharp8`, `Latest` or some other option that would automatically take care of testing with each and every new version starting at 8.

Reply:
`CSharp8` is good. `Latest` should never be used anywhere (it's not a version).